public/js/utils.js

```diff
@@ -104,3 +83,14 @@ var transformProject = function (data){
   data.data = toolColl;
   return data;
 }
+/*
+/projects returns a Sakai like /direct feed
+utility below turns it into an array of objects with the same structure
+as the CPM feeds of /migrating and /migrations for ease of comparing the three
+*/ 
+var transformMigrations = function (data){
+  $.each(data.data.entity, function(i, item){
+    item.tool_site_id= item.site_id + item.tool_id;
```

Comment:
Is it OK to concatenate site_id and tool_id directly and without a char in between?